lib/users.rb

```diff
@@ -0,0 +1,26 @@
+require 'rubygems'
+require "data_mapper"
+
+if ENV['ENVIRONMENT'] == 'test'
+  DataMapper.setup(:default, 'postgres://localhost/chitter_test')
+else
+  DataMapper.setup(:default, 'postgres://localhost/chitter')
+end
+
+class Users
+
+  include DataMapper::Resource
+
+  property :id,         Serial
+  property :name,       String
+  property :username,   String
+  property :email,      String
+  property :password,   String
+end
+
+DataMapper.finalize
```

Comment:
It feels like this code and the lines below belong in a more generic place, app.rb or a separate db_setup.rb - particularly as it is duplicated above in your chitter.rb.